src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import { FC } from 'react';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  const [searchParams] = useSearchParams();
+
+  return (
+    <Link
+      to={{
+        pathname: `/people/${person.slug}`,
+        search: searchParams.toString(),
+      }}
+      className={person.sex === 'f' ? 'has-text-danger' : ''}
```

Comment:
it's better to use classnames in cases like that, it allows you to dynamically toggle classes based on state or props without manually concatenating strings, making the code more maintainable and easier to read.